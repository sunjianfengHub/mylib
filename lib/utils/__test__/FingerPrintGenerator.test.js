// jest.mock('../Des.js');

let Fp;
beforeEach(() => {
	Fp = require('../FingerPrintGenerator.js');
})

describe('FingerPrint TestCase', () => {
	test('should work with des Properly', () => {
		expect(Fp.getFP('a')).toMatchSnapshot();
	})
	test('should not change for different import', () => {
		expect(Fp.getFP('a')).toMatchSnapshot();
	})
})