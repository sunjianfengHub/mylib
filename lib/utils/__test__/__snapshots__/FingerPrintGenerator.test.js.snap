// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FingerPrint TestCase should not change for different import 1`] = `
Array [
  "E238E4412DC93CC8EBE26313",
  "8CD5BA402A80FB5F4BB3C991",
]
`;

exports[`FingerPrint TestCase should work with des Properly 1`] = `
Array [
  "E238E4412DC93CC8EBE26313",
  "8CD5BA402A80FB5F4BB3C991",
]
`;
